src/components/content/order/create/GoToSubmit.tsx

```diff
@@ -37,7 +39,8 @@ export default function GoToSubmit({
             selectServiceHostingType,
             selectArea,
             selectRegion,
-            selectFlavor
+            selectFlavor,
+            selectContactServiceDetails
```

Comment:
this variable should not have the word "select" because the user does not select it.